src/Hng.Infrastructure/Context/ApplicationDbContext.cs

```diff
@@ -34,5 +39,8 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
         public DbSet<Notification> Notifications { get; set; }
         public DbSet<Comment> Comments { get; set; }
         public DbSet<Role> Roles { get; set; }
+        public DbSet<RolePermission> RolePermissions { get; set; }
+        public DbSet<UserRole> UserRoles { get; set; }
```

Comment:
No need having this here. EF will create it when you specify the relationship between entities

Reply:
need them to easily add or grant roles in controllers